source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
+	unlock: override func
+}
+
+MutexGlobal: class extends Mutex {
+	_globalMutex := static Mutex new(MutexType Safe)
+	init: func
+	lock: override func {
+		This _globalMutex lock()
+	}
+	unlock: override func {
+		This _globalMutex unlock()
+	}
+}
```

Comment:
It would be nice to have a cleanup function here, in case, for whatever reason (valgrind logs or who knows what), someone wants to be able to destroy the `_globalMutex`.